org.eclipse.xtend.core/xtend-gen/org/eclipse/xtend/core/macro/TransformationContextImpl.java

```diff
@@ -106,8 +106,8 @@ public List<? extends Problem> getProblems(final Element element) {
     return this.getProblemSupport().getProblems(element);
   }
 
-  public void validateLater(final Procedure0 validationCallback) {
```

Comment:
To be honest, I don't know where these changes in the generated code come from.

Reply:
@szarnekow  Just a quick note about this, because I still haven't reviewed everything: this happens to me as well when you run the maven build and use the xtend maven plugin. If you run clean build from eclipse the parameters go back to their original names. I never investigated further. Maybe it's due to some settings in this project that's only used by maven.

Reply:
There is an open issue for this. but of course I cannot find it

Reply:
I'm aware of that workaround, but it doesn't work anymore. Maybe tycho always runs the generator now. Not sure.

Reply:
@szarnekow I meant: if you rely on xtend from eclipse in the workspace it will keep the parameter names. If you run the maven build it will generate the parameter names with "arg".

Reply:
Yes, I understood what you were saying. But a clean-build in Eclipse doesn't fix it for me on a freshly oomphed Eclipse.

Reply:
Oh! That's strange.. and you have a jdk, not only a jre?

Reply:
I started looking into this: https://github.com/eclipse/xtext/pull/2995

Needs a change in the test-infra for common.types, though.